.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: run-tests
+
+on: [push]
```

Comment:
Please also add the checks for pull_reuqest as it was before.

Reply:
Done!